CHANGELOG.md

```diff
@@ -331,7 +334,6 @@ No significant changes.
 
 - Shadow effect to using box-shadow instead of drop-shadow for performance on embedded environment
 - `sandstone/FixedPopupPanels` and `sandstone/PopupTabLayout` to disable left key handler to go to the previous panel in RTL locales
-- `sandstone/MediaPlayer.MediaControls` to show more components when a user flicks on action guide
```

Comment:
I don't think we should remove previous released changelog

Reply:
fixed it